package.json

```diff
@@ -12,24 +12,24 @@
     "appsignal-diagnose": "./bin/diagnose"
   },
   "dependencies": {
-    "@opentelemetry/api": "^1.2.0",
-    "@opentelemetry/instrumentation-express": "^0.31.2",
-    "@opentelemetry/instrumentation-fastify": "^0.30.0",
-    "@opentelemetry/instrumentation-fs": "^0.5.1",
-    "@opentelemetry/instrumentation-graphql": "^0.31.0",
-    "@opentelemetry/instrumentation-http": "^0.32.0",
-    "@opentelemetry/instrumentation-ioredis": "^0.32.1",
-    "@opentelemetry/instrumentation-koa": "^0.33.0",
-    "@opentelemetry/instrumentation-mongodb": "0.32.2",
-    "@opentelemetry/instrumentation-mongoose": "^0.31.0",
-    "@opentelemetry/instrumentation-mysql": "^0.31.1",
-    "@opentelemetry/instrumentation-mysql2": "^0.32.0",
-    "@opentelemetry/instrumentation-nestjs-core": "^0.31.0",
-    "@opentelemetry/instrumentation-pg": "^0.31.1",
-    "@opentelemetry/instrumentation-redis": "^0.33.0",
-    "@opentelemetry/instrumentation-redis-4": "^0.33.0",
-    "@opentelemetry/sdk-node": "^0.33.0",
-    "@opentelemetry/sdk-trace-base": "^1.7.0",
+    "@opentelemetry/api": "^1.3.0",
```

Comment:
By bumping up the `@opentelemetry/api` version to `1.3.0`, [`@opentelemetry/api-metrics` is new deprecated](https://www.npmjs.com/package/@opentelemetry/api-metrics) and is no longer needed as a dependency. Could you remove it from the `overrides` section below?